### PR TITLE
Batch norm refactoring

### PR DESCRIPTION
**Pull request by OmarJabri7**
# PR Description 
a fix to reduce overhead when transpiling modules that use batch_norm, by adding copy=False to astype in order to remove any cloning 
Close #

## Checklist 

- [ x] Did you add a function?

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:
